doc/requirements-docs.txt

```diff
@@ -1,11 +1,30 @@
-# these are most specified that usual, because Sphinx docs seem to be quite fragile
```

Comment:
I'd leave the explanation of why in the comment. that is useful.

Reply:
Done!